docs/useful_views_new.md

```diff
@@ -0,0 +1,388 @@
+# Useful Views
+
+This page provides an overview of the most commonly used views in AUI Framework, each with a screenshot and a short description. Click on a view name to see its detailed documentation.
+
+<div class="views-grid">
+
+
+
+<!-- A2FingerTransformArea -->
+<div class="views-card-outer">
+  <div class="views-card">
+    <img src="../imgs/Views/A2FingerTransformArea.png" alt="A2FingerTransformArea screenshot" onerror="this.src='../imgs/logo_black.svg'">
+  </div>
+  <div class="views-card-title">A2FingerTransformArea</div>
+  <div class="views-card-desc">Handles and processes the multitouch scale, transform and rotation events based on two-finger gestures.</div>
+  <a class="views-card-link" href="#A2FingerTransformArea">Learn more</a>
+</div>
```

Comment:
This page was generated by doxygen.py:73; can we use that code instead? Title section can be picked up from page title, the desc can be used from @brief section of the underlying page.